msgraph-mail/ruby/graphrubyv4/users/item/inference_classification/inference_classification_request_builder.rb

```diff
@@ -2,17 +2,19 @@
 require '../item'
 require './inference_classification'
 require 'microsoft_kiota_abstractions'
-require_relative '../../../models/microsoft/graph/inference_classification'
-require_relative './overrides/item/inference_classification_override_request_builder'
+require_relative '../../../models/inference_classification'
```

Comment:
Files in the ./models/microsoft/graph folder can go directly into ./models/ since the dependencies in the files under the ./user folder all specify ../../../models/*

Reply:
ty!